test/testSetup.ts

```diff
@@ -123,15 +125,33 @@ export class TestSetup {
     utilities.log(`${this.testSuiteSuffixName} - Starting createProject()...`);
 
     const workbench = await (await browser.getWorkbench()).wait();
-    this.prompt = await utilities.runCommandFromCommandPrompt(
-      workbench,
-      'SFDX: Create Project',
-      50
-    );
+
+    // If you are not in a VSCode project, the Salesforce extensions are not running
+    // Force the CLI integration extension to load before creating the project
+    await utilities.runCommandFromCommandPrompt(workbench, 'Developer: Show Running Extensions', 5);
+    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Create Project', 5);
+    await browser.keys(['Escape']);
+    await utilities.pause(1);
+    await browser.keys(['Escape']);
+
+    // Do not continue until we verify CLI Integration extension is present and running
+    let coreExtensionWasFound = false;
+    do {
+      await utilities.pause(10);
+
+      coreExtensionWasFound = await utilities.findExtensionInRunningExtensionsList(
+        workbench,
+        'salesforcedx-vscode-core'
+      );
+    }
+    while (coreExtensionWasFound === false);
+    utilities.log(`${this.testSuiteSuffixName} - Ready to create the standard project`);
+
+    this.prompt = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Create Project', 5);
     // Selecting "SFDX: Create Project" causes the extension to be loaded, and this takes a while.
 
     // Select the "Standard" project type.
-    await utilities.selectQuickPickWithText(this.prompt, 'Standard');
+    await utilities.selectQuickPickItem(this.prompt, 'Standard');
```

Comment:
This is the only place where selectQuickPickWithText() is used.  The only difference between selectQuickPickWithText() and selectQuickPickItem() is that we can see the word "Standard" being typed out before it is selected.  Making this replacement therefore contains very minimal change in behavior, and E2E tests are still able to be run successfully.